docs/ensembl-help/ensembl-data/annotation-and-prediction/community-annotation.md

```diff
@@ -0,0 +1,45 @@
+---
+slug: community-genome-annotation
+title: Community genome annotation
+description: Requirements for community annotations to be imported into Ensembl.
+---
+
+# Community annotation formatting
+
+Whilst we encourage our users, submitters and providers to keep submitting their assembly annotations to [INSDC](https://www.insdc.org/about-insdc/), we can understand that this process may take longer than desired, delaying the timelines of when the genome will be available in our site. Thus, we exceptionally accept to get GFF3 files handed over directly to us (although the assembly will still need to be already publicly available in INSDC). We have put together a series of steps that can be easily followed to make sure a GFF3 file is valid and complies with our requirements.
```

Comment:
Not sure about the use of 'exceptionally' here, it doesn't quiet fit imo

Reply:
Changed the text slightly. Does it sound better now?

Reply:
I think im just unfamiliar with the use of this word in this context. I think thats fine now sorry